openmicroscopy/openmicroscopy
# Insight cleanup

Fix all warning noticed when building the javadoc

To test:
 * make sure you do not see any warning when the javadoc for insight is built.

cc @sbesson
